tensorrt_llm/_torch/disaggregation/native/transfer.py

```
@@ -549,6 +567,29 @@ def _deliver_kv_to_agent(self, write_meta: WriteMeta):
+                        # to its own allocated block count via
+                        # min(numBlocks, allocatedBlocks.size()).
+                        num_blocks = max(
+                            (len(ids) for ids in task._slice.block_ids_per_layer_groups),
+                            default=0,
+                        )
+                        session._on_chunk_transferred(
+                            request_id=session.request_id,
+                            chunk_block_offset=task._slice.chunk_block_offset,
+                            num_blocks=num_blocks,
+                        )
+                    except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)